cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java

```diff
@@ -31,6 +31,7 @@
 public class CartoDBBackendImpl extends HttpBackend implements CartoDBBackend {
     
     private final String apiKey;
+    private final boolean isPersonal;
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1393/commits/f2d7e43266253d141ea7e0f83ade64436c238e59